tests/bluetooth/controller/common/src/helper_util.c

```diff
@@ -190,10 +191,13 @@ void test_print_conn(struct ll_conn *conn)
 	printf("     Reject: %d\n", conn->llcp.remote.reject_opcode);
 	printf("--------------------->\n");
 }
-void test_setup(struct ll_conn *conn)
+void test_setup(struct ll_conn **conn)
```

Comment:
There should no longer be a need to do this as the buffer allocation freeeing no longer needs a conn handle and therefor `ll_conn_acquire()` below is no longer needed.

Reply:
Well, not really - the release still needs the conn object to keep track